lib/Test/Nginx/Util.pm

```diff
@@ -212,7 +212,9 @@ sub gen_rand_port (;$$) {
     my $rand_port;
 
     for (my $i = 0; $i < $tries; $i++) {
-        my $port = int(rand 63550) + 1985;
+        # NB: reserved ports for stream_server_config* (1..3)
+        # 1984 + 3 + 1 = 1988
+        my $port = int(rand 63550) + 1988;
```

Comment:
the port should not be greater than 65535.
so it is better to rand (63550 - 1998)